mmv1/third_party/terraform/resources/resource_google_service_account.go

```diff
@@ -115,7 +115,22 @@ func resourceGoogleServiceAccountCreate(d *schema.ResourceData, meta interface{}
 		return fmt.Errorf("Error reading service account after creation: %s", err)
 	}
 
-	return resourceGoogleServiceAccountRead(d, meta)
+	err = resourceGoogleServiceAccountRead(d, meta)
```

Comment:
We probably want to do structure this the same as `PollingWaitTime{{Resource}}PollRead` in MMv1. That'll automatically give us exponential backoff up to the time limit specified by the user), rather than a single retry like we have here (and as @slevenick points out below, may not be sufficient time).

Most notably, that will ensure that we verify the resource is provisioned over multiple calls, as a single success may not indicate it has propagated enough.

Reply:
Makes sense to me ! Thanks for the reference